projects/packages/my-jetpack/_inc/data/constants.ts

```diff
@@ -32,3 +32,21 @@ export const QUERY_PURCHASES_KEY = 'purchases';
 export const QUERY_EVALUATE_KEY = 'evaluate site recommendations';
 export const QUERY_SAVE_EVALUATION_KEY = 'save site evaluation result';
 export const QUERY_REMOVE_EVALUATION_KEY = 'remove site evaluation result';
+
+// Product Slugs
+export const PRODUCT_SLUGS = {
+	ANTI_SPAM: 'anti-spam',
+	BACKUP: 'backup',
+	BOOST: 'boost',
+	CRM: 'crm',
+	CREATOR: 'creator',
+	EXTRAS: 'extras',
+	JETPACK_AI: 'jetpack-ai',
+	SCAN: 'scan',
+	SEARCH: 'search',
+	SOCIAL: 'social',
+	SECURITY: 'security',
+	PROTECT: 'protect',
+	VIDEOPRESS: 'videopress',
+	STATS: 'stats',
+} satisfies Record< string, JetpackModule >;
```

Comment:
Learned new Typescript syntax today, thanks 😁

Reply:
I also learned that yesterday 😅 It seems like the best way to enforce the value of the object, but infer the types of the key. That way when you're using this in a typescript file, you will be limited to certain keys and still be able to see the value of the key 😄

Reply:
The other implementations I tried either had the key as a generic "string" type which is problematic as you could use any key you wanted without a typescript error, or the key would be defined but the value would be a generic "string" type and I wanted it to be linked to the JetpackModules type